cassandra-sink/README.adoc

```diff
@@ -1,7 +1,59 @@
+----
+CREATE KEYSPACE clouddata WITH REPLICATION = { 'class' : 'org.apache.cassandra.locator.SimpleStrategy', 'replication_factor': '1' } AND DURABLE_WRITES = true;
+USE clouddata;
+CREATE TABLE book  (
+    id          uuid PRIMARY KEY,
+    isbn        text,
+    author      text,
+    title       text
+);
+----
+
+You can then send data to this stream via
+
+  dataflow:>http post --contentType 'application/json' --data '{"isbn": "1599869772", "title": "The Art of War", "author": "Sun Tzu"}' --target http://localhost:8888/
+  > POST (application/json;charset=UTF-8) http://localhost:8888/ {"isbn": "1599869772", "title": "The Art of War", "author": "Sun Tzu"}
+  > 202 ACCEPTED
+
+and see the table contents using the CQL
+
+  SELECT * FROM clouddata.book;
 
 This module writes the content of each message it receives to Cassandra.
 
+=== Options
+
+The **$$cassandra$$** $$sink$$ has the following options:
+
+$$compressionType$$:: $$the compression to use for the transport$$ *($$CompressionType$$, default: `NONE`, possible values: `NONE,SNAPPY`)*
+$$consistencyLevel$$:: $$the consistencyLevel option of WriteOptions$$ *($$ConsistencyLevel$$, no default, possible values: `ANY,ONE,TWO,THREE,QUOROM,LOCAL_QUOROM,EACH_QUOROM,ALL,LOCAL_ONE,SERIAL,LOCAL_SERIAL`)*
+$$spring.cassandra.contactPoints$$:: $$the comma-delimited string of the hosts to connect to Cassandra$$ *($$String$$, default: `localhost`)*
+$$entityBasePackages$$:: $$the base packages to scan for entities annotated with Table annotations$$ *($$String[]$$, default: `[]`)*
+$$ingestQuery$$:: $$the ingest Cassandra query$$ *($$String$$, no default)*
+$$spring.cassandra.initScript$$:: $$the path to file with CQL scripts (delimited by ';') to initialize keyspace schema$$ *($$String$$, no default)*
+$$spring.cassandra.keyspace$$:: $$the keyspace name to connect to$$ *($$String$$, default: `<stream name>`)*
+$$metricsEnabled$$:: $$enable/disable metrics collection for the created cluster$$ *($$boolean$$, default: `true`)*
+$$spring.cassandra.password$$:: $$the password for connection$$ *($$String$$, no default)*
+$$spring.cassandra.port$$:: $$the port to use to connect to the Cassandra host$$ *($$int$$, default: `9042`)*
+$$queryType$$:: $$the queryType for Cassandra Sink$$ *($$Type$$, default: `INSERT`, possible values: `INSERT,UPDATE,DELETE,STATEMENT`)*
+$$retryPolicy$$:: $$the retryPolicy  option of WriteOptions$$ *($$RetryPolicy$$, no default, possible values: `DEFAULT,DOWNGRADING_CONSISTENCY,FALLTHROUGH,LOGGING`)*
+$$statementExpression$$:: $$the expression in Cassandra query DSL style$$ *($$String$$, no default)*
+$$spring.cassandra.schemaAction$$:: $$schema action to perform$$ *($$SchemaAction$$, default: `NONE`, possible values: `CREATE,NONE,RECREATE,RECREATE_DROP_UNUSED`)*
+$$ttl$$:: $$the time-to-live option of WriteOptions$$ *($$int$$, default: `0`)*
+$$spring.cassandra.username$$:: $$the username for connection$$ *($$String$$, no default)*
+
+//end::ref-doc[]
+
 == Requirements
```

Comment:
Requirements sections should be removed. We decided to only keep
* intro text
* Options (level 2)
* any other relevant examples, explanations, etc (without references to dataflow)
* Building (level 2, not in ref-doc)
* Running (level 2, not in ref-doc)